### PR TITLE
Fix #198 - Toolbar blocks header in mobile devices

### LINKED ISSUE #198: Toolbar blocks header in mobile devices

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: components/toolbars/ToolBars.vue [hunk 1]
```diff
@@ -16,13 +33,18 @@
...
+      <loading-toolbar
+        :isLoading="getIsLoading"
+        :isInTransaction="getIsInTransaction" />
+
       <tx-toolbar
         v-if="getPendingTx"
-        :txHash="getPendingTx"
-        :txInfo="getPendingTxInfo"
+        :txHash="'getPendingTx'"
+        :txInfo="'getPendingTxInfo'"
```
Comment (williamchong007) [inline components/toolbars/ToolBars.vue]: 'getPendingTxInfo' will become the string `getPendingTxInfo` instead of object?

Comment (nwingt) [reply components/toolbars/ToolBars.vue]: @williamchong007 Fixed

File: components/toolbars/ToolBars.vue [hunk 2]
```diff
@@ -62,6 +85,13 @@ export default {
     ChromeDialog,
     TxDialog,
   },
+  head() {
+    return {
+      htmlAttrs: {
+        'lc-toolbar': true ? 'show' : 'hidden',
```
Comment (williamchong007) [inline components/toolbars/ToolBars.vue]: 呢個 always `true` 點解要用 ternary operator？

Comment (nwingt) [reply components/toolbars/ToolBars.vue]: @williamchong007 Removed, not in use anymore

**Pull request merged**